RELEASE-NOTES.txt

```diff
@@ -2,7 +2,11 @@
 
 20.4
 -----
-
+* [*] Block Editor: Add React Native FastImage [https://github.com/WordPress/gutenberg/pull/42009]
```

Comment:
I think this should be an internal note, since it's a library addition and not a user-facing change. What do you think?

Reply:
Thank you for flagging that! Updated in https://github.com/wordpress-mobile/WordPress-Android/pull/16931/commits/efbf61b193f562cd9ef31ae4070a52c41a920396.